lib/service_extensions.dart

```diff
@@ -11,12 +11,16 @@ class ServiceExtensionDescription {
     this.extension,
     this.description,
     this.icon,
+    this.enabledValue = true,
+    this.disabledValue = false,
     String tooltip,
   }) : tooltip = tooltip ?? description;
```

Comment:
you should be able to add `assert(enabledValue is String || enabledValue is bool || enabledValue is num)`
right before the tooltip = tooltip ?? description; 
line.

Reply:
Gives error `Initializer expressions in constant constructors must be constants`